src/lib/transition_handler/processor.ml

```diff
@@ -93,7 +93,9 @@ module Make (Inputs : Inputs.S) :
                        Transition_frontier.add_breadcrumb_exn frontier
                          breadcrumb ;
                        Writer.write processed_transition_writer transition ;
-                       Catchup_scheduler.notify catchup_scheduler ~transition
+                       Deferred.return
```

Comment:
The original implementation of this was write. We would prefer `let%map` over `let%bind` if the statement does a monad.return